cg/services/orders/utils.py

```diff
@@ -4,5 +4,7 @@
 
 def parse_order(order: DatabaseOrder) -> PydanticOrder:
```

Comment:
Oof, I'd not name this `PydanticOrder` 😢  But stick to it if you want. If it is necessary to have a different name than `Order`, I'd add a `Dto` or something to it

Reply:
Changed it to Order, and we'll see if anyone complains in the future.